packages/client/docusaurus/docs/client/05-advanced/03-permissions-and-capabilities.mdx

```diff
@@ -3,4 +3,147 @@ id: 03-permissions-and-capabilities
 title: Permissions and Capabilities
 ---
 
-Stay tuned for more information on this topic.
+import SDKSpecific from '../SDKSpecific';
+
+## Introduction
+
+In some types of calls, there is a common requirement for participant moderation.
+A typical example is a webinar where the host wants to control who can speak or who can share their video or screen.
+
+Stream Video SDK provides a certain set of permissions and capabilities
+that can be used to control the behavior of participants in a call.
```

Comment:
```suggestion
The Stream Video SDK provides a certain set of permissions and capabilities
that can be used to control the behavior of participants in a call.
```

Reply:
Also: is it a pre-built set of permissions and capabilities? Can it be expanded to meet users' needs if they require more control?

Reply:
I think the set of permissions and capabilities is finite and can be assigned to a specific user role or call type. Roles and Call types would be eventually configurable on our Dashboard.